mappings/net/minecraft/client/render/model/json/JsonUnbakedModel.mapping

```diff
@@ -56,25 +59,28 @@ CLASS net/minecraft/class_793 net/minecraft/client/render/model/json/JsonUnbaked
 		ARG 3 side
 		ARG 4 settings
 		ARG 5 id
+	METHOD method_35789 needsResolution ()Z
 	CLASS class_795 Deserializer
+		FIELD field_32794 DEFAULT_AMBIENT_OCCULSION Z
+			COMMENT The default value for ambient occulsion if unspecified in JSON; is {@value}.
```

Comment:
```suggestion
		FIELD field_32794 DEFAULT_AMBIENT_OCCLUSION Z
			COMMENT The default value for ambient occlusion if unspecified in JSON; is {@value}.
```

Reply:
Ah, have been always calling it incorrectly and wondering why it's named in such a "weird" way :laughing: